src/Library/Lod/Internal/LodDirectoryHeader.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <stdint.h>
+
+
+#pragma pack(push, 1)
+struct LodDirectoryHeader_Mm6 {
+    inline LodDirectoryHeader_Mm6() {
+        memset(this, 0, sizeof(this));
+    }
+
+    uint8_t filename[16];
```

Comment:
I suggest we use `std::array` everywhere.

Reply:
i do fread/memset on this struct
will it work as expected?

Reply:
Yes, `std::array` is just a POD

Reply:
thanks, done